### ISSUE TITLE
editor can't read savefile

### ISSUE THREAD

**Issue by zerokos** [daidodo/diablo_edit#10]
when loading this savefile , the editor quit immediately without any info
but the game Diablo II: Resurrected can load the savefile without any trouble
so pls check it
[ck.zip](https://github.com/daidodo/diablo_edit/files/7377879/ck.zip)

**Comment by daidodo** [comment]
Thanks for the feedback!

Basically those characters are heavily modified so I can't guarantee the editor can load them.

For *ck.d2s*, there is a Town Portal Scroll ("tsc ") at an invalid position in belt . Though D2R can load it, the item will be strip away after saving. And the rings and amulet have 15 sockets with 7 jewels.

For *FALCON-DEMON.d2s*, there is a small charm with invalid property. I tested in D2R and it crashed when hovering the mouse on the item:
![image](https://user-images.githubusercontent.com/8170176/138184054-bc9501b7-4535-418c-92ca-31ceca2871f7.png)

For *PVc.d2s*, the problem is the right ring has 7 jewals, and one of them has an invalid property, too.

In summary, I can make a change to support 7 sockets. But for invalid position and property, I can't do anything.
Hope that can help you!

**Comment by zerokos** [comment]
ok，i got it ，thanks
